### ISSUE TITLE
S3 864 and high memory  issue under Windows 3.11

### ISSUE THREAD

**Issue by Fenix770** [86Box/86Box#1446]
**Describe the bug**
I found that after installing a 386 memory manager, Windows freezes with only an intermittent cursor on the left of the screen, 50% of the time. That means, for example, it freezes at first attempt, but works the second time after a hard reset, or after exit to DOS and attempt to load Windows for the second time it freezes again.

I have 3 copies of the same HD image:

1. I have a clean copy without any memory manager installed (Except EMM386 with /NOEMS parameter). This works fine.
2. A copy with QEMM 8.03, which has the bug.
3. And a copy with 386MAX v8 with the same problem despite to being a different software 

**To Reproduce**
1. Just start the VM
2. Type Win at DOS prompt $P$G
3. The machine frezzes at the first attempt.

Here is the NVR+CFG file: 

[486 FIC-IO2 VLB.zip](https://github.com/86Box/86Box/files/6524497/486.FIC-IO2.VLB.zip)

Here is a link with to the HD image file in a 7z  file format with a size of 125mb.
https://drive.google.com/file/d/1p-pdxn5IOEX2uCY6Prr753A103yFP2CQ/view?usp=sharing

Here is another image file with same size but with 386MAX installed instead of QEMM
https://drive.google.com/file/d/1w2PhLagmaNuLIS9BRlzfe6UUDMSSPyQL/view?usp=sharing

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - 86Box version: 3.0.0.2988
 - Build type: Dev

**Comment by OBattler** [comment]
Your link to the NVR+CFG is wrong.

**Comment by Fenix770** [comment]
I uploaded the NVR and CFG files again. It seems to be not uploaded right.

**Comment by OBattler** [comment]
It might be a conflict between your memory manager and the S3 driver you're using.

**Comment by Fenix770** [comment]
I also happen with the standard VGA driver included in Windows, and also that does not explain why it works after hard reset, after the first attempt to start Windows at DOS prompt. These software in theory only maps the unused regions of upper memory.

![Image 2](https://user-images.githubusercontent.com/31600563/119774082-0437bc00-be98-11eb-9825-c9d5eda6d732.png)

![Image 3](https://user-images.githubusercontent.com/31600563/119774086-06017f80-be98-11eb-82d9-fb5c424aea70.png)

**Comment by OBattler** [comment]
Does this still happen?

**Comment by Fenix770** [comment]
I think I found the reason of the issue. I think, the Afterdark screen saver is not compatible with QEMM in a same way as the CACHE86 software. I also found that CACHE86 and QEMM "Quick Boot" feature are incompatible with hard disk overlays. I use the HD overlay for the Mylex 486 machine for example.

![Afterdark](https://user-images.githubusercontent.com/31600563/133715917-bd02b1c9-dbb9-4d01-bb22-852af8a77800.png)
